package.json

```diff
@@ -143,7 +143,7 @@
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
     "select2": "^4.0.5",
-    "styled-components": "^3.2.6",
+    "styled-components": "^4.1.3",
```

Comment:
Should be the same `styled-components` version as https://github.com/Yoast/javascript/pull/184